source/runbooks/rotating-secrets.html.md.erb

```diff
@@ -1,25 +1,33 @@
 ---
 owner_slack: "#modernisation-platform"
 title: How to Rotate Secrets
-last_reviewed_on: 2023-08-17
+last_reviewed_on: 2023-11-22
 review_in: 6 months
 ---
 
 # <%= current_page.data.title %>
 
 ## Introduction
 
-We don't have many secrets on stored on the Modernisation Platform, but they are rotated regularly.
+We don't have many secrets on stored on the Modernisation Platform, but they are rotated regularly, every 180 days.
```

Comment:
We don't have many secrets on stored on the Modernisation Platform, but ones deemed 'secret' are rotated every 180 days.

Reply:
This was already added so not sure why it didn't show in here.